CONTRIBUTING.md

```diff
@@ -6,21 +6,26 @@ We're happy to help [semgrep@r2c.dev](mailto:semgrep@r2c.dev)
 
 ## Development Workflow
 
-Before each commit,
-git should run [`pre-commit`](https://pre-commit.com/)
-to ensure files are well-formatted
-and check for basic linting bugs.
-If you don't have `pre-commit` installed,
-the following command will do so for you:
+Pull requests require approval of at least one maintainer and [CI to be passing](https://github.com/returntocorp/semgrep-rules/actions).
+
+Getting a new rule ready for review is easy to do. The CI jobs will check for these things:
+1. Formatting applied by [`pre-commit`](https://pre-commit.com/)
+1. Passing tests
+
+In addition, we recommend:
+1. Having a good rule message. A good message includes a description of the pattern, a description of why this pattern was detected, and an alternative that resolves the issue.
+1. Using namespacing for your rule.
```

Comment:
What does "Using namespacing for your rule." mean?

Reply:
Oh I see below. Should we make these bullets links to the headers below?

Reply:
Yeah, that makes sense 👍